ci/install-cloud-sdk.sh

```diff
@@ -26,7 +26,11 @@ ARCH="$(uname -p)"
 if [[ "${ARCH}" == "aarch64" ]]; then
   # The tarball uses this name
   ARCH="arm"
-fi
+else 
+  # For x86_64, uname -p might give unknown 
+  ARCH="$(uname -m)"
```

Comment:
Thanks.  Sounds like we should simply change line 25?

Reply:
How would I verify this commands works as intended on the ARM architecture?

Reply:
We don't have a standard answer for this, or even good non-standard answers (this is a gap in our automated testing). One approach is to spin up a T2A VM:

https://cloud.google.com/compute/docs/machine-resource#arm

Another approach would be to run a Docker container with qemu.  I have read that is doable, don't know how to make it work.

Reply:
Ok I'll look into both